api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -424,6 +424,14 @@ message HttpConnectionManager {
   // Note that Envoy does not perform
   // `case normalization <https://tools.ietf.org/html/rfc3986#section-6.2.2.1>`
   google.protobuf.BoolValue normalize_path = 30;
+
+  // Determines if adjacent slashes in the path are merged into one before any processing of
+  // requests by HTTP filters or routing. This affects the upstream *:path* header as well. Without
+  // setting this option, incoming requests with path `//dir/file` will not match against route with
```

Comment:
Mind 
//dir/file -> //dir///file
to make it clear that it isn't just the first /// being elided?